crates/wasmi/src/engine/config.rs

```diff
@@ -83,32 +88,6 @@ impl Config {
         features
     }
 
-    /// Sets the [`StackLimits`] for the [`Config`].
-    pub fn set_stack_limits(&mut self, stack_limits: StackLimits) -> &mut Self {
-        self.stack_limits = stack_limits;
-        self
-    }
-
-    /// Returns the [`StackLimits`] of the [`Config`].
-    pub(super) fn stack_limits(&self) -> StackLimits {
-        self.stack_limits
-    }
-
-    /// Sets the maximum amount of cached stacks for reuse for the [`Config`].
-    ///
-    /// # Note
-    ///
-    /// Defaults to 2.
-    pub fn set_cached_stacks(&mut self, amount: usize) -> &mut Self {
-        self.cached_stacks = amount;
-        self
-    }
-
-    /// Returns the maximum amount of cached stacks for reuse of the [`Config`].
-    pub(super) fn cached_stacks(&self) -> usize {
-        self.cached_stacks
-    }
-
```

Comment:
Please rebase your work onto `main`. These methods already do not exist on `main`.